android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIBottomNavigationTabGroup.java

```diff
@@ -362,6 +362,12 @@ public void updateTabBackgroundDrawable(int index)
 		}
 	}
 
+	@Override
+	public void updateActiveIndicatorColor(int color)
+	{
+
+	}
```

Comment:
Override w/o implementation?

Reply:
This is the old/current BottomNavigation. I use this method just in the new version but since all extend from the TiUIAbstractTabGroup I had to add an empty method in there.

Reply:
Even if unused? I wonder why thats the case

Reply:
it is the new `indicatorColor` :

![Screenshot_20250312-222119](https://github.com/user-attachments/assets/c57ea4fa-c67b-4381-aca5-e1debb37ccad)

to set the background color of the M3 selection.